### PR TITLE
May news

### PR DESCRIPTION
**Pull request by maelle**

### CODE REVIEW

File: content/blog/2023-05-19-ropensci-news-digest-may-2023/index.Rmd [hunk 1]
```diff
@@ -0,0 +1,658 @@
...
+You can read this post [on our blog](`r url`).
+Now let's dive into the activity at and around rOpenSci!
+
+## rOpenSci HQ
+
+### Stat software review
+
+TODO Mark ;-)
+
+### rOpenSci at csvconf
```
Comment (yabellini) [inline content/blog/2023-05-19-ropensci-news-digest-may-2023/index.Rmd:47]:
```suggestion
### rOpenSci at Csv,Conf,v7
```

File: content/blog/2023-05-19-ropensci-news-digest-may-2023/index.Rmd [hunk 2]
```diff
@@ -0,0 +1,658 @@
...
+
+TODO Yani ;-)
```
Comment (yabellini) [inline content/blog/2023-05-19-ropensci-news-digest-may-2023/index.Rmd:49]:
```suggestion
We participated in the [CSV,Conf,v7](https://csvconf.com/) in Buenos Aires during April 19-20, 2023.

On the first day of the conference, Karthik Ram was one of the keynotes presenting ["How to enable and sustain thriving Open Source Ecosystems (OSE)."](/events/2023-04-19-csvconf_keynote/).  The next day, Yanina Bellini Saibene presented ["Tell me who you hang out with, and I will tell you who you are. A collaborations analysis using social networks analysis"](/events/2023-04-20-csvconf-regulartalk/), a work done together with Sandro Camargo. 

This was a very special edition as it was the first time the event took place in the Southern Hemisphere, allowing several Latin American projects to be part of the conference.
```

**Comment by maelle** [comment]
thanks @yabellini!!

**Review by steffilazerte — COMMENTED** [review]
Looks good. Always love the Dev corner, I feel like this is the main area I get tips and tricks now :grin: 

Just a couple of suggestions, other than that, definitely good to go. Let me know if you want me to make a final check after Mark's additions.

* [X] post follows Style Guide
* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md
* [X] YAML subject tags are ok ("tech notes" for tech notes; "community" for non-staff non-editor)
* [X] YAML field 'preface' is present if necessary

File: content/blog/2023-05-19-ropensci-news-digest-may-2023/index.md [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+
+* Four at ['3/reviewer(s)-assigned'](https://github.com/ropensci/software-review/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A3/reviewer(s)-assigned):
+
+     * [pangoling](https://github.com/ropensci/software-review/issues/575), Access to Large Language Model Predictions. Submitted by [Bruno Nicenboim](https://bnicenboim.github.io/). 
+
+    * [ohun](https://github.com/ropensci/software-review/issues/568), Optimizing Acoustic Signal Detection. Submitted by [Marcelo Araya-Salas](https://marceloarayasalas.weebly.com/). 
+
+    * [dfms](https://github.com/ropensci/software-review/issues/556), Dynamic Factor Models. Submitted by [Sebastian Krantz](https://github.com/SebKrantz). 
+
+    * [NA](https://github.com/ropensci/software-review/issues/546), NA. Submitted by [Alexander Fischer](https://s3alfisc.github.io/blog/).  (Stats).
```
[the remark this answers is not in the document]
Comment (maelle) [reply content/blog/2023-05-19-ropensci-news-digest-may-2023/index.md:110]: ah yes, there was a mix up in that submission between "Description" and the full DESCRIPTION, now fixed, thank you!!

File: content/blog/2023-05-19-ropensci-news-digest-may-2023/index.md [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+
+* Two at ['2/seeking-reviewer(s)'](https://github.com/ropensci/software-review/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A2/seeking-reviewer(s)):
+
+     * [mregions2](https://github.com/ropensci/software-review/issues/590), Access Data from Marineregions.org: The Marine Regions Gazetteer and the Marine Regions Data Products. Submitted by [salvafern](https://github.com/salvafern). 
+
+    * [bssm](https://github.com/ropensci/software-review/issues/489), Bayesian Inference of Non-Linear and Non-Gaussian State Space. Submitted by [Jouni Helske](https://jounihelske.netlify.app).  (Stats).
+
+* One at ['1/editor-checks'](https://github.com/ropensci/software-review/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A1/editor-checks):
+
+     * [qualtdict](https://github.com/ropensci/software-review/issues/572), Generating Variable Dictionaries and Labelled Data Exports of Qualtrics. Submitted by [lyh970817](https://github.com/lyh970817).
```
[the remark this answers is not in the document]
Comment (maelle) [reply content/blog/2023-05-19-ropensci-news-digest-may-2023/index.md:120]: correct, I use the GitHub name but can fix it here.

**Pull request merged**